druid/src/command.rs

```diff
@@ -73,19 +122,42 @@ enum Arg {
     OneShot(Arc<Mutex<Option<Box<dyn Any>>>>),
 }
 
-/// Errors that can occur when attempting to retrieve the a command's argument.
+/// Errors that can occur when attempting to retrieve the a `OneShotCommand`s argument.
 #[derive(Debug, Clone, PartialEq)]
 pub enum ArgumentError {
-    /// The command did not have an argument.
-    NoArgument,
-    /// The argument was expected to be reusable and wasn't, or vice-versa.
-    WrongVariant,
-    /// The argument could not be downcast to the specified type.
-    IncorrectType,
+    /// The command represented a different selector.
+    WrongSelector,
     /// The one-shot argument has already been taken.
     Consumed,
 }
 
+/// This error can occure when wrongly promising that a type ereased
```

Comment:
Because the user still has to 'promise' that any menus or windows they create match the application state. This error occurs when the promise is broken.

Reply:
oh this comment was out of date, sorry